RecoBTag/ImpactParameter/python/pfNegativeOnlyJetProbabilityBJetTags_cfi.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+pfNegativeOnlyJetProbabilityBJetTags = cms.EDProducer("JetTagProducer",
```

Comment:
I'm not sure it's particularly practical to define so many almost identical cfi files.
If you decide to add some common parameters to JetTagProducer, the maintenance will be more costly than having one default and cloning with replace from it.